pandas/io/formats/style_render.py

```diff
@@ -98,6 +98,8 @@ def __init__(
 
         # add rendering variables
         self.hidden_index: bool = False
+        self.hidden_rows: Sequence[int] = []
```

Comment:
can you change the impl to use the same nomenclature, e.g. hidden_index, hidden_columns (or similar)

Reply:
Can you give your input to the suggestion below:
```
self.hide_index_: bool
self.hide_columns_: bool
self.hidden_indexes: Sequence[int]
self.hidden_columns: Sequence[int]
```

Reply:
updated (used rows instead of indexes)